src/install.js

```diff
@@ -492,10 +492,8 @@ async function createInitialBug() {
     console.log('Creating initial bug!');
 
     await Bugs.post({
-        bid: 1,
```

Comment:
Why is this bid removed...?

Reply:
The db keeps track of the `nextBid` to be created, so there's no need to hardcode it here